src/components/authentication/CreateProfilePage.tsx

```diff
@@ -0,0 +1,225 @@
+import * as ImagePicker from "expo-image-picker";
+
+import {
+  CreateProfilePageRouteProp,
+  CreateProfilePageNavigationProp,
+} from "../../routes";
+import Text from "../themed/Text";
+import Box from "../themed/Box";
+import ThemedButton from "../themed/ThemedButton";
+import ThemedInput from "../themed/ThemedInput";
+
+// React Navigation Types and Page Options
+
+type CreateProfilePageProps = {
+  route: CreateProfilePageRouteProp;
+  navigation: CreateProfilePageNavigationProp;
+};
+
+// Note: Needed to create this as a its own component to access navigation in the header
+const SkipButton = () => {
+  const navigation = useNavigation();
+  return (
+    <Box me="lg">
+      <TouchableOpacity
+        onPress={() => {
+          navigation.reset({
+            index: 0,
+            routes: [{ name: "Login" }],
+          });
+        }}
+      >
+        <Text variant="link">Skip</Text>
+      </TouchableOpacity>
+    </Box>
+  );
+};
+
+export const CreateProfilePageOptions = {
+  title: "",
+  headerTransparent: true,
+  headerRight: () => <SkipButton />,
+};
+
+// Page Styles
+
+const styles = StyleSheet.create({
+  root: {
+    flex: 1,
+    flexDirection: "column",
+    justifyContent: "flex-start",
+    alignItems: "center",
+  },
+  form: {
+    width: "100%",
+    flexGrow: 1,
+  },
+});
+
+// Generic Image Selection
+
+// Since there are two types of permissions and ways to select images (but the rest of the code is the same)
+// I created a way to generify the approach by allowing the permission method and image selection as function args
+type PermissionMethod = () => Promise<ImagePicker.PermissionResponse>;
+type SelectionMethod = (
+  options?: ImagePicker.ImagePickerOptions | undefined
+) => Promise<ImagePicker.ImagePickerResult>;
+
+// Options to be used for selecting the image, currently is 1:1 and allows cropping
+const imagePickerOptions: ImagePicker.ImagePickerOptions = {
+  mediaTypes: ImagePicker.MediaTypeOptions.Images,
+  allowsEditing: true,
+  aspect: [1, 1],
+  quality: 1,
+};
+
+export default function CreateAccountPage({
+  navigation,
+}: CreateProfilePageProps) {
+  // Hook used to show and hide the bottomsheet for image selection
+  const [isBottomSheetVisible, setBottomSheetVisible] = useState(false);
+  // Hook used to store local image url for profile image
+  const [profileImageURL, setProfileImageURL] = useState("");
+  // Generic method that allows user to select an image from the gallery/camera after requesting permissions
+  // TODO: Add dialog/alert if user denies permission
+  const selectImage = async (
+    permissionMethod: PermissionMethod,
+    selectionMethod: SelectionMethod
+  ) => {
+    const { status } = await permissionMethod();
+    if (status === ImagePicker.PermissionStatus.GRANTED) {
+      const result = await selectionMethod(imagePickerOptions);
+      if (!result.cancelled) {
+        setProfileImageURL(result.uri);
+      }
+    }
+    setBottomSheetVisible(false);
+  };
+
```

Comment:
maybe make this its own component as well. If we decide to allow communities to have an image then this code can be reused